packages/lesswrong/lib/modules/fragments.js

```diff
@@ -232,6 +232,9 @@ registerFragment(`
 registerFragment(`
   fragment SelectCommentsList on Comment {
     ...CommentsList
+    parentComment {
```

Comment:
This will significantly increase our loading times on the frontpage, since the query will only resolve after the parent has been loaded. This seems fine for now, but eventually it seems better to query for the parent comment when you click the parent comment button.

Reply:
But this does make the button snappier, so not sure what the best thing to do is.